modules/appeals_api/app/swagger/higher_level_reviews/v0/swagger.json

```diff
@@ -16,6 +16,92 @@
   ],
   "paths": {
     "/forms/200996": {
+      "get": {
+        "summary": "List Higher-Level Reviews",
```

Comment:
I just reviewed our current Decision Reviews and segmented (SC, NOD, HLR) documentation, and noticed that this would be the first `index` endpoint that we document. Our addition of `index` endpoints predates my time on the team, so I'm probably missing context because of that, but was there a reason why we had some `index` endpoints before but didn't document them? Were they intended to be used to our team only, or had we just not gotten as far as documenting them?